src/promptflow-tools/promptflow/tools/azure_content_safety.py

```diff
@@ -0,0 +1,319 @@
+import enum
```

Comment:
no need?

Reply:
This is align with sample code of Azure Content Safety GA version.